src/main/java/org/apache/maven/plugin/surefire/report/ConsoleTreeReporter.java

```diff
@@ -62,21 +63,21 @@ public void testSetStarting( TestSetReportEntry report )
     @Override
     public void testSetCompleted( WrappedReportEntry report, TestSetStats testSetStats, List<String> testResults )
     {
-
         for ( WrappedReportEntry testResult : testSetStats.getReportEntries() )
         {
             final  MessageBuilder builder = buffer().a( "| " + TEST_SET_STARTING_PREFIX );
+            String testResultShort = StringUtils.abbreviate(StringUtils.normalizeSpace(testResult.getReportName()), "\u2026", 180);
```

Comment:
Hey! Can you please:
- Use constants for the code and number of characters?
- Make use of static import for `StringUtils`?